README.md

```diff
@@ -1 +1,115 @@
 # OKRs 2018 Q4
+
+## Bring 'Code as Data' to market with enterprises
+
+### Ensure gitbase interoperability with 3rd party tools [DR]
+
+- [ ] Drivers for in-house languages (Python, Go, Java).
+- [ ] mysqldump
+- [ ] grafana
+- [ ] Tableau
+- [ ] Drivers for other popular languages (JavaScript, Ruby, .NET, PHP).
+
+### Get gitbase enterprise ready [DR]
+
+- [ ] Authentication and authorization abstractions prepared for future integration with LDAP and Kerberos
+- [ ] Basic MySQL auth implementation
+- [ ] Audit logs (connections, queries)
+
+### Get web interfaces enterprise ready [APPS]
+
+- [ ] gitbase-web SSO (protocol TBD)
+- [ ] bblfsh-web SSO (protocol TBD)
+
+### Gitbase research [DR]
+
+- [ ] Initial research on supporting updates in backing data (indexes, etc)
+- [ ] Speed up retrieving UASTs
+
+### Launch gitbase-spark-connector [DR]
```

Comment:
Is there a design document about these connectors?
Do we have an alpha version available somewhere?

Reply:
Sent links through Slack. We are working on the first version (private), which will be splitted into community/enterprise versions this quarter.